ricecooker/classes/nodes.py

```diff
@@ -823,12 +830,6 @@ def to_dict(self):
             "extra_fields": json.dumps(self.extra_fields),
             "role": self.role,
             "suggested_duration": self.suggested_duration,
-            "grade_levels": self.grade_levels,
-            "resource_types": self.resource_types,
-            "learning_activities": self.learning_activities,
-            "accessibility_categories": self.accessibility_labels,
-            "subjects": self.categories,
-            "needs": self.learner_needs,
```

Comment:
These still need to be here - the `to_dict` method doesn't inherit currently, so it needs to repeat what `TreeNode` does.